flambe/runner/run.py

```diff
@@ -93,7 +97,7 @@ def main(args: argparse.Namespace) -> None:
                 else:
                     raise ValueError("Only ClusterRunnables can be executed in a cluster.")
         else:
-            runnable, _ = ex.preprocess(secrets=args.secrets, install_ext=args.install_extensions)
+            runnable, _ = ex.preprocess(secrets=args.secrets, install_ext=args.install_extensions, debug=args.debug)
```

Comment:
I would prefer the `debug` parameter to be passed to the `run` method of the runnables instead of injecting it through the `preprocess`.

This would involve changing the API of runnable to receive this extra parameter (defaults to `False`) in its `run` method.